test/integration/controlplane/transformation/transformation_test.go

```diff
@@ -286,14 +288,18 @@ func (e *transformTest) getRawSecretFromETCD() ([]byte, error) {
 }
 
 func (e *transformTest) getEncryptionOptions(reload bool) []string {
+	var options []string
 	if e.transformerConfig != "" {
-		return []string{
+		options = append(options, []string{
 			"--encryption-provider-config", filepath.Join(e.configDir, encryptionConfigFileName),
 			fmt.Sprintf("--encryption-provider-config-automatic-reload=%v", reload),
-			"--disable-admission-plugins", "ServiceAccount"}
+			"--disable-admission-plugins", "ServiceAccount"}...)
+	}
+	if e.enableRBAC {
+		options = append(options, "--authorization-mode=RBAC")
```

Comment:
let me know RBAC to be enabled for all existing tests by default

Reply:
@tkashem yeah, just enable it always.  The diff should be small enough to simply include in #127513

Reply:
yes, enabled, and the tests are green so far.